infrastructure/kube/templates/keep-ecdsa/initcontainer/provision-keep-ecdsa/provision-keep-ecdsa.js

```diff
@@ -207,28 +206,11 @@ async function authorizeSortitionPoolContract(operatorAddress, sortitionPoolCont
   console.log(`Authorized!`)
 }
 
-async function createSortitionPool(applicationAddress) {
-  const ADDRESS_ZERO = "0x0000000000000000000000000000000000000000"
-
+async function getSortitionPool(applicationAddress) {
   let sortitionPoolContractAddress
-
-  const create = async () => {
-    await bondedECDSAKeepFactory.methods.createSortitionPool(applicationAddress).send({ from: contractOwnerAddress })
-
-    console.log(`created sortition pool for application: [${applicationAddress}]`)
-
-    return await bondedECDSAKeepFactory.methods.getSortitionPool(applicationAddress).call()
-  }
-
+  
   sortitionPoolContractAddress = await bondedECDSAKeepFactory.methods.getSortitionPool(applicationAddress).call()
```

Comment:
We can combine lines 210-212:
```suggestion
const sortitionPoolContractAddress = await bondedECDSAKeepFactory.methods.getSortitionPool(applicationAddress).call()
```